npm/fake_nsp_package.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
```

Comment:
I think this is no longer in use anywhere since in system tests I wrote this part as well.